churchland_pipeline/acquisition.py

```diff
@@ -71,9 +156,61 @@ class Problem(dj.Part):
         ---
         problem_cause: varchar(255) # (e.g. corrupted data)
         """
+        
+    
+    def quickinsert(monkey, rig, task):
+        """
+        Quickly insert session data
+        """
+        
+        # check keys
+        assert len(lab.Monkey & {'monkey': monkey})==1, 'Unrecognized monkey'
+        assert len(lab.Rig & {'rig': rig})==1,          'Unrecognized rig'
+        assert len(Task & {'task': task})==1,           'Unrecognized task'
+
+        # session dates
+        rawPath = Session.rawpath(rig,task,monkey)
+        dates = sorted(list(os.listdir(rawPath)))
+        dates = [x for x in dates if re.search('\d{4}-\d{2}-\d{2}',x) is not None]
+
+        # import session data (can make task-specific later)
+        for date in dates:
+            
+            sessFiles = os.listdir(rawPath + date + '/')
+            
+            # this will need to be updated for non-blackrock data files
+            if (not any(Session & {'session_date': date})
```

Comment:
```suggestion
            if (not Session & {'session_date': date}
```
DataJoint tables provide a `bool` function, which return `True` it's non-empty.